dev-client/src/components/buttons/icons/common/InfoButton.tsx

```diff
@@ -15,17 +15,20 @@
  * along with this program. If not, see https://www.gnu.org/licenses/.
  */
 
-import {forwardRef} from 'react';
-import {PressableProps, View} from 'react-native';
-
 import {IconButton} from 'terraso-mobile-client/components/buttons/icons/IconButton';
+import {InfoSheet} from 'terraso-mobile-client/components/sheets/InfoSheet';
 
-type InfoButtonProps = {
-  onPress?: PressableProps['onPress'];
-};
+type InfoButtonProps = React.PropsWithChildren<{
+  sheetHeading?: React.ReactNode;
+}>;
 
-export const InfoButton = forwardRef<View, InfoButtonProps>(
-  ({onPress}: InfoButtonProps, ref) => (
-    <IconButton ref={ref} type="sm" name="info" onPress={onPress} />
-  ),
+export const InfoButton = ({
```

Comment:
This was combined with the old `InfoOverlaySheetButton` to be just a single component, since there were no uses of `InfoButton` without an attached sheet and the `IconButton` component is much more standardized now.